src/apps/integrations/loris/domain/loris_integrations.py

```diff
@@ -1,29 +1,24 @@
-import uuid
+import json
 
 from apps.integrations.db.schemas import IntegrationsSchema
-from apps.integrations.domain import AvailableIntegrations
 from apps.shared.domain import InternalModel
 
 
-class IntegrationMeta(InternalModel):
+class LorisIntegration(InternalModel):
     hostname: str
     username: str
     password: str
-    project: str | None
-
-
-class Integrations(InternalModel):
-    applet_id: uuid.UUID
-    type: AvailableIntegrations
-    configuration: IntegrationMeta
+    project: str
 
     @classmethod
     def from_schema(cls, schema: IntegrationsSchema):
-        return cls(applet_id=schema.applet_id, type=schema.type, configuration=schema.configuration)
-
-
-class IntegrationsCreate(InternalModel):
-    applet_id: uuid.UUID
-    hostname: str
-    username: str
-    password: str
+        new_loris_integration_dict = json.loads(schema.configuration.replace("'", '"'))
+        new_loris_integration_dict["password"] = "*****"
```

Comment:
Minor nit: You could add a new class (example `LorisIntegrationResponse`) that omits this value from the response, and use it for `retrieve_integration`. Similar approach would be needed for `FutureIntegration` to omit `api_key` since we're not showing that value.

Reply:
The right use of this is always use "PublicModel" classes to input and output and "InternalModel" classes to process data, this way we can control what to received (and validate) and what to output